### ISSUE TITLE
Unable to start the webpage

### ISSUE THREAD

**Issue by pippo73** [sungshon/PimpMyStremio#80]
On my win10 pc, I start the app, it updates everything, and I find the icon on the traybar.
But when I select one of the options ("open in browser", "load in stremio web", or "load in stremio app" , nothing happens...

do I miss something?

**Comment by BurningSands70** [comment]
I recently got a similar issue on my own app that also uses the `open` module to open links programmatically.

@sungshon My theory is that the `open` module broke on Win after some update, I'll try updating the module in my app to see if that fixes it. You should probably try updating it too.

**Comment by sungshon** [comment]
@BurningSands70 This is a strange issue, I've been unable to reproduce it, I also noticed that you use `shell.openExternal` from Electron in your project, not the `open` module. I think something else might be breaking this.. but I can't be sure of what.. the `open` module doesn't seem to have any specific commit related to fixing this in their history.

**Comment by BurningSands70** [comment]
@sungshon 

Yeah, I noticed I was using `shell.openExternal()` after I posted, I'm also suspecting something else.. maybe firewall or AV blocking the web pages opening?

If I ever figure it out, I'll tell u too.
